### ISSUE TITLE
Typecho独立页面使用报错

### ISSUE THREAD

**Issue by XenWayne** [Drew233/Artitalk#28]
**描述**
Typecho 主题使用了pjax.
主题链接:https://github.com/shiyiya/typecho-theme-sagiri/tree/pjax

报错信息:
Argument 1 passed to DOMNode::replaceChild() must be an instance of DOMNode, null given

**出现bug的页面**
页面链接：https://xenwayne.top/test.html
在得到回复前，我将会一直保留此页面。

**截图**
如果方便的话，在这里添加截图来更清晰的说明你遇到的bug。

**桌面设备 (please complete the following information):**
 - 操作系统: [Windows]
 - 浏览器 [edge]
 - 版本 [84.0.522.52

**Comment by Drew233** [comment]
你好，是否方便提供一下你新建的单独页面内容写的是什么的截图？

**Comment by XenWayne** [comment]
> 你好，是否方便提供一下你新建的单独页面内容写的是什么的截图？


（因为主题已经加载jquery，所以实际上我省去了引入jquery这一行。）
在独立的html中运行正常:https://xenwayne.top/words.html


以下是内容:

!!!
<html>
	<body>
                <script src="https://libs.baidu.com/jquery/2.0.0/jquery.min.js"></script>	
		<script>
		var appID="******";
		var appKEY="******";
		</script>
		<div id="artitalk_main"></div>
		<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/artitalk"></script> 
	</body>
	<style>
	 /*artitalk修改样式*/		
	.cbp_tmtimeline>li:nth-child(odd) .cbp_tmlabel {
	  background: linear-gradient(60deg, rgba(255, 165, 150, 0.5) 5%, rgba(0, 228, 255, 0.35)) 0% 0% / cover, url("https://ae01.alicdn.com/kf/H18a4b998752a4ae68b8e85d432a5aef0l.png"), url("https://ae01.alicdn.com/kf/H21b5f6b8496141a1979a33666e1074d9x.jpg") 0px 0px;
	  color: white;
	}
	.cbp_tmtimeline>li:nth-child(odd) .cbp_tmlabel:after {
	  border-right-color:  rgba(255, 165, 150, 0.5);
	}
	.cbp_tmtimeline>li .cbp_tmlabel {
	  background: linear-gradient(60deg, rgba(255, 165, 150, 0.5) 5%, rgba(0, 228, 255, 0.35)) 0% 0% / cover, url("https://ae01.alicdn.com/kf/H18a4b998752a4ae68b8e85d432a5aef0l.png"), url("https://ae01.alicdn.com/kf/H21b5f6b8496141a1979a33666e1074d9x.jpg") 0px 0px;
	  color: white;
	}
	.cbp_tmtimeline>li .cbp_tmlabel:after {
	  border-right-color:  rgba(0, 228, 255, 0.35);
	}
	.button {
	  background: linear-gradient(60deg, rgba(255, 165, 150, 0.5) 5%, rgba(0, 228, 255, 0.35)) 0% 0% / cover, url("https://ae01.alicdn.com/kf/H18a4b998752a4ae68b8e85d432a5aef0l.png"), url("https://ae01.alicdn.com/kf/H21b5f6b8496141a1979a33666e1074d9x.jpg") 0px 0px;
	  color: white;
	}
	</style>	
</html>
!!!

**Comment by Drew233** [comment]
你试一下把`<html>`标签去掉，如果还是不可以的话，是不是主题不支持单独页面使用html？

**Comment by XenWayne** [comment]
> 你试一下把`<html>`标签去掉，如果还是不可以的话，是不是主题不支持单独页面使用html？

去掉标签后是新的报错信息:smile:
https://xenwayne.top/test.html

如果是主题的问题，我单独写一个html好了:smile:

**Comment by Drew233** [comment]
我现在进不去你的链接欸，科学上网也不行

如果可以的话写一个html也可以😭

等我有时间用你的主题复现一下问题试试哈

**Comment by Drew233** [comment]
滴滴， 我这里刚才测试了一下你这个主题，是可以的欸
http://47.98.185.31/index.php/shuoshuo.html
是不是你修改了一些本地文件导致配置出现一些小错误之类的？

**Comment by XenWayne** [comment]
> 滴滴， 我这里刚才测试了一下你这个主题，是可以的欸
> http://47.98.185.31/index.php/shuoshuo.html
> 是不是你修改了一些本地文件导致配置出现一些小错误之类的？

感谢跟进~我给的链接是pjax的版本:joy:

**Comment by Drew233** [comment]
原来不是一个版本啊😂

**Comment by Drew233** [comment]
我用了pjax那个分支下的主题，还是可以鸭😂（可能是我主题又安装错了

如果找不到问题的话写一个html单页感觉也不错

**Comment by XenWayne** [comment]
> 我用了pjax那个分支下的主题，还是可以鸭😂（可能是我主题又安装错了
> 
> 如果找不到问题的话写一个html单页感觉也不错

嗯，感谢跟进，我去看看插件（
